Tella/Scenes/Resources/Views/Common/ResourceCard.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  ResourceCard.swift
+//  Tella
+//
+//  Created by gus valbuena on 2/2/24.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import SwiftUI
+
+struct ResourceCard: View {
+    var title: String
+    var serverName: String
+    var rightButtonImage: String
+    var rightButtonAction: () -> Void
+    
+    var body: some View {
+        VStack(spacing: 0) {
+            HStack {
+                Image("resources.pdf")
+                    .padding()
+                ConnectionCardDetail(title: title, subtitle: serverName)
+                Spacer()
+                MoreButtonView(imageName: rightButtonImage, action: rightButtonAction)
```

Comment:
I don't think that `MoreButtonView` is reusable component because it's a simple button, but if you see that i't reusable button with image you need to update the name for example : `ImageButtonView`